http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/ChannelAttributeKey.java

```diff
@@ -73,6 +73,12 @@ public final class ChannelAttributeKey {
     public static final AttributeKey<ChannelDiagnostics> CHANNEL_DIAGNOSTICS = NettyUtils.getOrCreateAttributeKey(
         "aws.http.nio.netty.async.channelDiagnostics");
 
+    /**
+     * {@link AttributeKey} to keep track of whether we have received the {@link LastHttpContent}.
+     */
+    public static final AttributeKey<Boolean> LAST_HTTP_CONTENT_RECEIVED_KEY = NettyUtils.getOrCreateAttributeKey(
```

Comment:
Should we rename it to `STREAMING_COMPLETE_KEY`? Alternatively, we can update the description of this field to make it clear it to indicate streaming has finished

Reply:
Done